src/python/pants/backend/python/targets/python_distribution.py

```diff
@@ -38,31 +36,23 @@ def __init__(self,
     :type payload: :class:`pants.base.payload.Payload`
     :param sources: Files to "include". Paths are relative to the
       BUILD file's directory.
-    :type sources: ``Fileset`` or list of strings. Must include setup.py.
-    :param compatibility: either a string or list of strings that represents
-      interpreter compatibility for this target, using the Requirement-style
-      format, e.g. ``'CPython>=3', or just ['>=2.7','<3']`` for requirements
-      agnostic to interpreter class.
+    :type sources: :class:`twitter.common.dirutil.Fileset` or list of strings. Must include
```

Comment:
Given that this changes the behaviour of sources, you might want to merge from master and re-test post #5908